Kickstarter-iOS/Views/Cells/DiscoveryProjectCardCell.swift

```diff
@@ -109,22 +154,25 @@ final class DiscoveryProjectCardCell: UITableViewCell, ValueCell {
 
     _ = self.projectNameLabel
       |> projectNameLabelStyle
+      |> UIView.lens.contentCompressionResistancePriority(for: .vertical) .~ .required
```

Comment:
On iPhone 5S the title and blurb labels would get clipped without this setting. Thought it would be good to specify it everywhere where it's really important.